examples/plotting/checkout_form.py

```diff
@@ -1,3 +1,12 @@
+'''This example shows a sample checkout form created using bokeh widgets.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.widgets.button, bokeh.models.widgets.checkbox,
+    bokeh.models.widgets.groups, bokeh.models.widgets.inputs
```

Comment:
The sampledata directive is not currently smart enough to handle multiple lines, these will need to be combined into one line
```suggestion
    :apis: bokeh.models.widgets.button, bokeh.models.widgets.checkbox, bokeh.models.widgets.groups, bokeh.models.widgets.inputs
```

Reply:
Yes sir will do the required changes.

Reply:
Sir, I'm having trouble in writing proper refs, could you please guide me about it.